### PR TITLE
I made a new change

### PR DESCRIPTION
**Pull request by ahmadourfali**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1 +1,2 @@
 hello world
+this the changes that I'm making
```
Comment (QuitHub) [inline index.html]: I think you should put this in html tags really!

**Review by QuitHub — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (ahmadourfali) [reply index.html]: ok I will

File: index.html [hunk 2]
```diff
@@ -1 +1,11 @@
-hello world
+!doctype <!DOCTYPE html>
+<html lang="en" dir="ltr">
```
Comment (QuitHub) [inline index.html:2]: what does this mean - `dir="ltr"`

File: index.html [hunk 3]
```diff
@@ -1 +1,11 @@
...
+  <head>
+    <meta charset="utf-8">
```
Comment (QuitHub) [inline index.html:4]: do you know what this means too?

File: index.html [hunk 4]
```diff
@@ -1 +1,11 @@
...
+    <title></title>
```
Comment (QuitHub) [inline index.html]: why no title?

**Review by QuitHub — APPROVED** [review]
the most import thing is the title is missing

**Comment by ahmadourfali** [comment]
Good morning mate,

I managed to do, not exactly what you asked me to do though.

**Comment by ahmadourfali** [comment]
lang stands for language which is English in this case.
dir stands for text direction which is left to right in this case. 
charset UTF-8 stands for the character set or character encoding system used. This is important so that the web browser knows how to display certain work. in this case I used UTF-8 which covers almost all of the characters and symbols in the world.

**Comment by ahmadourfali** [comment]
I also added a title to my html

File: index.html (hunk 2 above)
Comment (QuitHub) [reply index.html:2]: Reply like this (below!)

Comment (QuitHub) [reply index.html:2]: HERE!!!

Comment (QuitHub) [reply index.html:2]: lang stands for language which is English in this case.
dir stands for text direction which is left to right in this case.

**Comment by QuitHub** [comment]
Ok, looks good to me!

**Pull request merged**